src/components/playerOverview/CommanderMatchupsTable.tsx

```diff
@@ -21,9 +23,9 @@ export const CommanderMatchupsTable = React.memo(function CommanderMatchupsTable
     const navigate = useNavigate();
 
     // get all the matches of the player has participated in
```

Comment:
can we update this comment as well to match the other one in PlayerMatchupsTable?

Reply:
done.